agent/jvm/src/test/java/org/jolokia/jvmagent/security/DelegatingAuthenticatorTest.java

```diff
@@ -122,7 +122,7 @@ private HttpPrincipal executeAuthCheck(String pSpec) {
 
     @Test
     public void invalidProtocol() {
-        DelegatingAuthenticator authenticator = new DelegatingAuthenticator("jolokia","ftp://ftp.redhat.com",null,false);
+        DelegatingAuthenticator authenticator = new DelegatingAuthenticator("jolokia","http://ftp.redhat.com",null,false);
```

Comment:
The aim of the test is to test an invalid url, restricted url, or invalid protocol?

Reply:
I would say restricted URL since the assertion is done on the failure response 401
        assertEquals(failure.getResponseCode(),401);

Reply:
name of the method says protocol (ok, it's kindof minor)